recipes/emsdk_installer/all/conanfile.py

```diff
@@ -0,0 +1,121 @@
+from conans import ConanFile, tools
+import os
+
+
+class EmSDKInstallerConan(ConanFile):
+    name = "emsdk_installer"
```

Comment:
We don't append such packages with `_installer` anymore.
```suggestion
    name = "emsdk"
```

Reply:
Is it acceptable that this package downloads its own node.js? I think I can make this package pass the conan hooks and still work. But is this wanted?

Reply:
Shouldn't it be `emscripten`?